global.css

```diff
@@ -5,20 +5,20 @@ html#stylus #header *:not(#\1transition-suppressor) {
   transition: none !important;
 }
 body {
-  font: normal 12px Arial, system-ui, sans-serif;
+  font: normal .75rem sans-serif;
 }
 
 body:lang(ja) {
-  font-family: Arial, 'Meiryo UI', 'MS Gothic', system-ui, sans-serif;
+  font-family: sans-serif, 'Meiryo UI', 'MS Gothic', system-ui;
```

Comment:
It was the first thing I've tried but the appearance changed noticeably.

Reply:
If the interface is displayed in en then yes, all en glyph will not be rendered in `Arial` anymore but `Meiryo UI` or `MS Gothic`.

Reply:
No, I've tried in Japanese/Chinese, of course.

Reply:
I don't understand. AFAIK the first font will be used only for characters defined in that font so, assuming the default is Arial, nothing should change for CJK languages.

Reply:
Then that probably because you have a different default in ja/zh.

The default (fallback) font for CJK is different from other languages. Since its original purpose is to be used as fallback, it has to support a wide range of characters in the specific language. For example, [serif defaults to `MingLiU` in zh-tw](https://github.com/openstyles/stylus/pull/474#issuecomment-414234893).

If our goal is to provide a nice default interface, I think we should exclude CJK from this PR.

Reply:
Eh, for some reason I didn't see your latest comment so I was always replying to the pre-pre one.

Reply:
Can't we just remove these rules altogether? The purpose of that PR was to fix the problem caused by our explicit usage of `Arial`. As long as we just use `sans-serif` we shouldn't need any of these weird shit, just like no one else does.